webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/StatusReports.scala

```diff
@@ -76,7 +76,7 @@ final class RuleStatusReport private (
   , val overrides : List[OverridenPolicy]
 ) extends StatusReport {
   val compliance = report.compliance
-  val byNodes: Map[NodeId, AggregatedStatusReport] = report.reports.groupBy(_.nodeId).mapValues(AggregatedStatusReport(_))
+  val byNodes: Map[NodeId, AggregatedStatusReport] = report.reports.groupBy(_.nodeId).map{case (a, value) => (a, AggregatedStatusReport(value))}
```

Comment:
it *looks* like this files should not have been changed, as the cost for homepage skyrockets